tests/foreman/api/test_capsulecontent.py

```diff
@@ -1947,6 +1949,12 @@ def test_positive_read_with_non_admin_user(
 
         :customerscenario: true
         """
+        # Assign the capsule to the default org/loc so it can be searched and found.
+        nc = module_capsule_configured.nailgun_smart_proxy
+        target_sat.api.SmartProxy(
```

Comment:
This seems like an important prerequisite for any test, having capsule assigned to default org/loc. Should we maybe do that in [capsule_setup](https://github.com/SatelliteQE/robottelo/blob/01b794d76f504c3a78f45760ecba151f5b629427/robottelo/hosts.py#L1654)? Just a suggestion, could be done in a separate PR or in this one.

Reply:
@vsedmik what do you think? I can merge this PR as it is.

Reply:
@jameerpathan111 I thought the same - could be in the setup or the fixture, but wasn't sure if it could or should interfere with the other tests. Also there is a question why it doesn't happen on registration automatically.

This PR would just fix the test regardless what we find.

Reply:
Yeah, I was curious about why the capsule is not assigned to default org/loc. It's worth further investigation. Anyway, we can merge this PR.

Reply:
@jameerpathan111 @vsedmik I would lean against assigning the Capsule to the default org/loc in `capsule_setup`. I'm working on tests for the upgrade scenarios refactor that create fresh orgs/locs and assign Capsules to them directly without interacting with the default org/loc. While I don't think that having Capsules assigned to the default org/loc as well would interfere with this, I would prefer to maintain the option to isolate them.

I think it would make more sense to do this in one or more of the `capsule_configured` fixtures from `robottelo/pytest/fixtures/core/sat_cap_factory.py`

Reply:
@synkd Yup, that supports my reluctance to do so globally. Some `autouse` module-scoped fixture for that would be yet another way to go.